src/Repository/SearchesRepository.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Searches;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+use Psr\Cache\InvalidArgumentException;
+use Symfony\Contracts\Cache\CacheInterface;
+use Symfony\Contracts\Cache\ItemInterface;
+
+/**
+ * @extends ServiceEntityRepository<Searches>
+ *
+ * @method Searches|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Searches|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Searches[]    findAll()
+ * @method Searches[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class SearchesRepository extends ServiceEntityRepository
+{
+    private CacheInterface $myCachePool;
+
+    public function __construct(ManagerRegistry $registry, CacheInterface $topTagsCache)
+    {
+        $this->myCachePool = $topTagsCache;
+        parent::__construct($registry, Searches::class);
+    }
+
+    public function save(Searches $entity, bool $flush = false): void
+    {
```

Comment:
Repositiry should only have queries that return something.
I do not thing that you need methods like save and remove since you can use EntittyManager in Controller to save something.

Reply:
These methods are generated by Symfony automatically.

Ok, I will delete them.